### ISSUE TITLE
Add docs directory

### ISSUE THREAD

**Issue by chadestep** [surmeierlab/neurphys#4]
Need to add a docs directory and fill it with useful documents to aide in creating extensions of the package. I suggest modeling it after [NumPy](https://github.com/numpy/numpy/tree/master/doc)

**Comment by titipata** [comment]
There is an wiki page in the repository. Might be a good alternative to add simple documentation in markdown format.

**Comment by chadestep** [comment]
Yeah, Dan and I were having that exact same discussion yesterday. Still haven't come to a conclusion...

**Comment by titipata** [comment]
Yeah, some how it's better to have separate documentation, I totally agree.
